### ISSUE TITLE
Missing data between 2019-12-27 and 2020-08-25

### ISSUE THREAD

**Issue by GeorgeSG** [LubomirGeorgiev/bnb-currency-exchange-rates#1]
Hi,

Thanks for the repo, this is really useful!

I'm using the GBP and USD rates, and I noticed there's a big gap in the data for 2020. This applies to the top 14 currencies.

Here's an excerpt from one of the files:
```
"2020-08-27T00:00:00.000Z","2.18516","0.457632","GBP",
"2020-08-26T00:00:00.000Z","2.18066","0.458577","GBP",
"2020-08-25T00:00:00.000Z","2.17447","0.459882","GBP",
"2019-12-27T00:00:00.000Z","2.29746","0.435263","GBP",
"2019-12-26T00:00:00.000Z","2.28197","0.438218","GBP",1
"2019-12-25T00:00:00.000Z","2.28197","0.438218","GBP",1
```

I went back in the history of the files and it seems like this has always been a problem.

**Comment by GeorgeSG** [comment]
After some digging, the Start Year for this request is set to 2020 instead of 2019:

```ts
console.log(`START: ${new Date(periodStart)}`);
console.log(`FORMATTED: ${formatDate(new Date(periodStart), 'Y')}`);
```

Outputs:

```
Period Start: 2019-12-30T15:36:48.620Z Period End: 2020-08-25T14:36:48.620Z
START: Mon Dec 30 2019 17:36:48 GMT+0200 (Eastern European Standard Time)
FORMATTED: 2020
index.htm?periodStartDays=30&periodStartMonths=12&periodStartYear=2020&periodEndDays=25&periodEndMonths=8&periodEndYear=2020&type=XML&downloadOper=true&group1=second&search=true&showChart=false&showChartButton=true&valutes=USD
```

**Comment by LubomirGeorgiev** [comment]
Fixed in #2
